data/language/en-GB.yml

```diff
@@ -2261,3 +2261,12 @@ strings:
   2206: "+10% everywhere"
   2207: "Min everywhere"
   2208: "Max everywhere"
+  2209: "All vehicles"
+  2210: "Stopping at station"
+  2211: "Transporting cargo"
+  2212: "{MOVE_X 10}Carrying {STRINGID} {SPRITE}"
+  2213: "»{MOVE_X 10}Carrying {STRINGID} {SPRITE}"
```

Comment:
Not too sure about the wording here. I thought 'Transporting' was too long, but perhaps just 'Load of' would be better.

Reply:
I guess technically trains could have less than 1 Load of a cargo.  Maybe "Contains cargo"?

Reply:
Hmm, "Contains cargo" is even longer. To be honest, I think we should probably just drop the verb entirely.